app/src/main/java/com/example/androidacademyhw/details/FragmentMoviesDetails.kt

```diff
@@ -10,19 +10,25 @@ import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
 import com.example.androidacademyhw.FragmentClickListener
 import com.example.androidacademyhw.R
 import com.example.androidacademyhw.Tags
 import com.example.androidacademyhw.data.Actor
 import com.example.androidacademyhw.data.Movie
+import com.example.androidacademyhw.data.loadActors
+import com.example.androidacademyhw.data.loadMovies
 import com.example.androidacademyhw.databinding.FragmentMoviesDetailsBinding
+import kotlinx.coroutines.*
 
-class FragmentMoviesDetails : Fragment()  {
+class FragmentMoviesDetails : Fragment() {
     private var fragmentClickListener: FragmentClickListener? = null
 
     private var _binding: FragmentMoviesDetailsBinding? = null
     private val binding get() = _binding!!
 
+    private var coroutineScope = CoroutineScope(Dispatchers.IO + SupervisorJob())
```

Comment:
Also u need cancel `coroutineScope `in `onDestroy `callback for example.

Reply:
Done